object/slicer/slicer.go

```diff
@@ -140,18 +151,34 @@ func (x *Slicer) childPayloadSizeLimit() uint64 {
 // '__NEOFS__' prefix.
 //
 // See New for details.
-func (x *Slicer) Slice(data io.Reader, attributes ...string) (oid.ID, error) {
+func (x *Slicer) Slice(header object.Object, data io.Reader, attributes ...string) (oid.ID, error) {
```

Comment:
why do we have both `header` and `attributes` in that commit?

Reply:
Made a few rebases/reverts and misses this moment. Updated